src/lightning_app/core/constants.py

```diff
@@ -35,7 +35,7 @@ def get_lightning_cloud_url() -> str:
 REDIS_QUEUES_READ_DEFAULT_TIMEOUT = 0.005
 
 HTTP_QUEUE_URL = os.getenv("LIGHTNING_HTTP_QUEUE_URL", "http://localhost:9801")
-HTTP_QUEUE_REFRESH_INTERVAL = float(os.getenv("LIGHTNING_HTTP_QUEUE_REFRESH_INTERVAL", "0.1"))
+HTTP_QUEUE_REFRESH_INTERVAL = float(os.getenv("LIGHTNING_HTTP_QUEUE_REFRESH_INTERVAL", "1"))
```

Comment:
Not clear to me what the practical implication of setting this to 1 is. 1 second is a noticeable lag so we should be careful here

Reply:
Only the works with the `caller_queue` are waiting infinitely.  Delaying the work to start 1 later in the cloud should be fine. This would reduce drastically the number of calls.